plugins/in_tail/tail_config.h

```diff
@@ -83,9 +83,9 @@ struct flb_tail_config {
     int refresh_interval_sec;  /* seconds to re-scan           */
     long refresh_interval_nsec;/* nanoseconds to re-scan       */
     int read_from_head;        /* read new files from head     */
-    int rotate_wait;           /* sec to wait on rotated files */
+    time_t rotate_wait;        /* sec to wait on rotated files */
```

Comment:
No, this is not what I meant, I didn't ask you to change more things, I asked you to change less things, please rever this.

Reply:
Also, `rotate_wait` and `ignore_older` are both defined as `FLB_CONFIG_MAP_TIME` in the configuration map.

Please don't change anything else until we are sure we understand each other and see the purported issues from the same perspective, otherwise this makes the review process harder and in turn the PR less viable.